src/components/Lesson/ImprovePage.scss

```diff
@@ -81,25 +38,34 @@
 }
 
 /* Student / Teacher mode */
+
+/* Box shadow inset used to highlight box so it differentiates better from other objects */
 .improvePageBoxStudent {
   composes: improvePageBox;
   background-color: $color-green-light;
+  box-shadow: inset 0 0 85px #89c785;
```

Comment:
Burde disse fargene vært definert i variables.scss?

Når det er sagt, så må jeg innrømme at jeg synes improvePage-boxen blir litt mørk med denne box-shadow'en, og den føles litt "out of place" når resten av siden er "flat". Hvis det var opp til meg, ville jeg fjernet denne box-shadowen (og den for teacher også da, selvsagt). Hva tenker du om det?